free-1.md

```diff
@@ -10,41 +10,157 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Tableau trees is widely used in logic. 
+There is a LaTeX 
+[package that's specifically for drawing tableau trees](http://ctan.org/pkg/prooftrees)
+but its syntax is quite challenging.
+So this project would be for logicians who are not familiar enough with LaTeX
+to feel comfortable using the packages for drawing trees.
+Ideally, this project would eliminate the need to drawing out lines and moving
+text boxes using formatting tools like Google Slides, which is the recommended
+formatting tool in CS 81.
 
-### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+People already use a package in a DSL (LaTeX) to draw tableau trees and trees
+in general.
+But a tableau tree has enough distinguishing characteristics from any tree 
+diagram that a DSL for tableau trees should have a simplified interface.
+In CS 81, the construction of a tableau tree is reduced to two operations:
+stack and split. So I think a DSL can use that technique to build a simpler
+syntax for a DSL for tableau trees.
+A DSL for this would make the task much less cumbersome as hopefully users
+would find the DSL simpler to use than drawing out a tree using lines and text
+boxes.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I have to draw tableau trees for my CS 81 homework and they cannot be hand
+drawn.
+I generally TeX my homework, but I've found the syntax of packages for drawing
+trees to be difficult to understand.
+I imagine that the syntax would be even more daunting for people who do not
+use LaTeX for most of their homework.
+I TeX-ed tableau trees in the latest homework but it required a lot of trial
+and error as well as reading a _lot_ of samples of various tree structures.
+I would be really happy if there was an easier way for me to draw trees for 
+my homework.
```

Comment:
It might be interesting to allow the language to generate LaTeX code (for various packages/backends) if you just want to stick it into your existing LaTeX file.